OemPkg/OemPkg.dec

```diff
@@ -47,6 +47,13 @@
   # Include/Guid/PasswordStoreVariable.h
   gOemPkgPasswordStoreVarGuid =  {0xa2ee0f0b, 0xac46, 0x436e, {0xaf, 0xe6, 0x40, 0x60, 0xee, 0x63, 0xd6, 0xa2} }
 
+  # Oem Config Policy Guid
+  gOemConfigPolicyGuid = { 0xba320ade, 0xe132, 0x4c99, { 0xa3, 0xdf, 0x74, 0xd6, 0x73, 0xea, 0x6f, 0x76 } }
+
+[Ppis]

```

Comment:
Do we really need this?

Reply:
The PPI guid should be removed. The gOemConfigPolicyGuid we do need to publish the policy.